apps/web/pages/organizations/[organizationId]/invoices/[invoiceId]/components/invoiceDetails.tsx

```diff
@@ -231,7 +213,7 @@ export const InvoiceDetails = ({ invoice: invoiceFragment }: InvoiceDetailsProps
               Print
             </button>
           </div>
-          <div>
+          <div className="text-left">
```

Comment:
This doesn't seem to do anything for me
```suggestion
          <div>
```